app/views/users/show.html.haml

```diff
@@ -17,30 +17,40 @@
               %span.name
                 = @user.full_name
               %span.account @#{@user.nickname}
-              = render 'follow_relationships/follow_form'
+              = render 'follow_relationships/follow_form', user: @user
             .dates
               .blue-box
               %span Born #{@user.birthday.strftime('%m/%d/%Y')}
               .blue-box
               %span Joined #{@user.created_at.strftime('%m/%d/%Y')}
-          - if @user == current_user
+          - if @user.id == current_user.id
             .edit-profile
               = link_to 'Edit Profile', edit_user_path(id: @user.id), class: 'button no-fill-button'
         %nav
           #nav-tab{class: 'nav nav-tabs justify-content-center', role: 'tablist'}
             %button{class: 'nav-link active', id: 'nav-posts-tab', 'data-bs-toggle': 'tab', 'data-bs-target': '#nav-posts', role: 'tab', 'aria-controls': 'nav-posts', 'aria-selected': 'true'}
-              %p #{@user.posts.count.to_s}
-              Posts
+              %p #{@user.posts.count}
+              = @user.posts.count == 1 ? 'Post' : 'Posts'
```

Comment:
Take a look at the[ rails pluralize method](https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize)